abce/group.py

```diff
@@ -0,0 +1,42 @@
+class Group(object):
+    def __init__(self, sim, group, name):
+        self.sim = sim
+        self.group = group
+        self.name = name
+        self.do = self.execute_parallel if sim.processes > 1 else self.execute_serial
+
+    def __add__(self, g):
+        return Group(self.sim, self.groups + g.groups)
+
+    def execute_serial(self, command):
+        messages = self.sim.messagess
+        for family in self.group:
+            messages[family.name()] = family.execute(command, messages[family.name()])
+        messages[('_simulation', 0)] = []
+        messages[('_simulation', 0.5)] = []
+        self.sim._agents_to_add.extend(messages.pop(('_simulation', 0), []))
```

Comment:
I see now. L14 actually does include a message with the key ('_simulation', 0) and ('_simulation', 0.5), so L15 and L16 are possibly no longer necessary.